src/main/markdown/faq.md

```diff
@@ -0,0 +1,49 @@
+
+- [Lagom Gitter channel](https://gitter.im/lagom/lagom)
+- [lagom-framework mailing list](https://groups.google.com/forum/#!forum/lagom-framework)
+- [Stack Overflow Lagom questions](https://stackoverflow.com/questions/tagged/lagom)
+
+## Is Lagom open source or commercial?
+
+Lagom is completely open source, free to use, and has a vibrant and growing community. However, you do have the option to purchase a [Lightbend subscription](https://www.lightbend.com/platform/subscription) that provides technical support as well as license to use Lightbend Enterprise Suite, which includes a range of powerful application management, intelligent monitoring, and enterprise integration tooling.
+
+## Does Lagom require me to use Cassandra for persistence?
+
+No, Lagom supports using either Cassandra or relational databases, including MySQL, PostegreSQL, Oracle and H2. The Lagom development environment provides and uses Cassandra by default because it offers great scalability and replication features.  See more in the [Java documentation](/documentation/current/java/PersistentEntity.html) or the [Scala documentation](/documentation/current/scala/PersistentEntity.html).
+
+## Does Lagom require a me to run a Kafka cluster?
+
+No, but it is very useful for common communication scenarios. Lagom's Message Broker API integrates Persistent Entities with [Apache Kafka](https://kafka.apache.org/) to provide publish and subscribe messaging between services with at-least-once delivery guarantees. This enables you to design decoupled and highly-resilient systems of microservices.
+
+If you prefer not to run a Kafka cluster, however, the Message Broker API is completely optional. You can write Lagom services that communicate with each other directly by using HTTP requests or by streaming data over a WebSocket.
+
+The Message Broker API was designed to make it possible to support for alternative message broker services in a future version of Lagom.
+
+See more in the [Java documentation](/documentation/current/java/MessageBroker.html) or the [Scala documentation](/documentation/current/scala/MessageBroker.html).
+
+## Does Lagom require using event sourcing and CQRS?
+
+You can handle data persistence any way you like in a Lagom service, but we strongly encourage event sourcing and CQRS, and the Lagom Persistence API is designed to support these patterns. For services where you do not wish to use event sourcing and CQRS, you can execute arbitrary database statements using Lagom's `CassandraSession`, `JdbcSession` or `JpaSession`. You can also use any other data access client you choose, as long as you are careful to avoid I/O blocking the primary thread pool.
+
+See more in the Lagom documentation on [Managing data persistence](/documentation/current/java/ES_CQRS.html) and [Advantages of Event Sourcing](/documentation/current/java/ESAdvantage.html).
+
+## Can I serialization methods other than JSON?
```

Comment:
"Can I serialization methods other than JSON?"

"Can I (use) serialization methods other than JSON?"

Reply:
Fixed in a later commit

Reply:
Can I haz nice serialisations? :P